string board

int32 action
int32 UPDATE_AND_CHANGE_COLOR=0
int32 CHANGE_COLOR=1
int32 UPDATE_BUT_KEEP_COLOR=2

string key
string LOGGING_KEY=logging
string RECORDING_KEY=recording
string STREAMING_KEY=streaming
string SETUP_KEY=setup
string CONTROL_KEY=control

string PREDICTION_CURRENT_KEY=cur_pred
string PREDICTION_CURRENT_PROGRESS_KEY=cur_progress
string PREDICTION_CURRENT_FAILURE_KEY=cur_failure
string PREDICTION_CURRENT_SCORE_KEY=cur_score
string PREDICTION_SUCCESSOR_KEY=suc_pred
string PREDICTION_SUCCESSOR_PROGRESS_KEY=suc_progress
string PREDICTION_SUCCESSOR_FAILURE_KEY=suc_failure
string PREDICTION_SUCCESSOR_SCORE_KEY=suc_score
string PREDICTION_FILTERED_KEY=pred_filt
string PREDICTION_PROGRESS_FILTERED_KEY=pred_progress_filt

string DESCRIPTION_KEY=executing
string LEFT_PROGRESS_KEY=left_progress
string RIGHT_PROGRESS_KEY=right_progress

string value
string STARTING_VALUE=starting
string STARTED_VALUE=started
string STOPPING_VALUE=stopping
string STOPPED_VALUE=stopped
string FAILED_VALUE=failed

string INTERRUPTING_VALUE=interrupting
string CONTINUING_VALUE=continuing
string INTERRUPTED_VALUE=interrupted
string CONTINUED_VALUE=continued

string ACTIVE_VALUE=active
string FROZEN_VALUE=frozen

string ON_VALUE=on
string OFF_VALUE=off
string UNKNOWN_VALUE=unknown

int32 color
int32 WHITE=0
int32 GREEN=1
int32 YELLOW=2
int32 RED=3
int32 PURPLE=4
int32 BLUE=5
int32 NUM_COLORS=6